### PR TITLE
Test coverage

### PR DESCRIPTION
**Pull request by yuriyKolomiyets**
Slava, pls check it

### CODE REVIEW

File: src/test/java/TestSqlUserService.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    public TestSqlUserService() throws SQLException {
+    }
+
+    @Before
+    public void before() throws ClassNotFoundException, SQLException {
+
+        conn = DriverManager.getConnection(DB_URL, USER, PASS);
+        Class.forName("org.h2.Driver");
+
+        {
```
Comment (bizianov) [inline src/test/java/TestSqlUserService.java]: try/catch is redundant here. new SqlUserServiceImpl(conn) is not throwing any exception.

File: src/test/java/TestSqlUserService.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            try {
+                sqlUserService = new SqlUserServiceImpl(conn);
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+
+        Statement st = conn.createStatement();
```
Comment (bizianov) [inline src/test/java/TestSqlUserService.java]: where do you use it?

**Comment by bizianov** [comment]
Very good, code is clean and working, I left only 2 comments, but noticed a couple of minor things I want you to change. Sorry, for being so boring but I want you to adhere a good code practices from the very beginning:
1. Open each class and check if you have unused imports. If yes, use Ctrl+Alt+O combination.
2. Check if you have unused code, like static final String JDBC_DRIVER = "org.h2.Driver";
3. Remove redundant like breaks (in some places you have to many enters)

File: src/test/java/TestSqlUserService.java (hunk 2 above)
Comment (yuriyKolomiyets) [reply src/test/java/TestSqlUserService.java]: in the middle of nowhere))))

fixed

File: src/test/java/TestSqlUserService.java (hunk 1 above)
Comment (yuriyKolomiyets) [reply src/test/java/TestSqlUserService.java]: fixed

**Pull request merged**